src/planscape/forsys/views.py

```diff
@@ -14,6 +14,10 @@
                          JsonResponse, QueryDict)
 
 
+# Constants for spatial references.
+kSpatialReferenceForRasters = "+proj=aea +lat_0=23 +lon_0=-96 +lat_1=29.5 +lat_2=45.5 +x_0=0 +y_0=0 +datum=WGS84 +units=m +no_defs"
+kSpatialReferenceForHucBoundaries = 4269
```

Comment:
There's a config value for this in src/planscape/config/boundary.json. It might be cleaner to pull this from there!

Reply:
Done. Removed. I tried 2 solutions and went with the latter.

1) I pulled this from src/planscape/config/boundary.json.
2) I realized I was using 4269 to represent SRID's of project area polygons and boundary polygons, which could be different. I've opted to set project area SRID by hand - it should ultimately be an input parameter - and to access boundary polygon SRID directly from the db object.